denops/denippet/main.ts

```diff
@@ -228,6 +228,24 @@ export function main(denops: Denops): void {
       return parsed;
     },
 
+    async snippetIdToString(idU: unknown): Promise<string> {
+      const id = u.ensure(idU, is.String);
+      const parsed = lsputil.parseSnippet(id);
```

Comment:
A snippet ID is UUID.
https://github.com/uga-rosa/denippet.vim/blob/c0a74b0f84069bbd73f4676da72e3920d9560f39/denops/denippet/loader.ts#L100-L102

It should be the body of the snippet that is passed to `parseSnippet()`.

Reply:
```typescript
    async snippetIdToString(idU: unknown): Promise<string> {
      const id = u.ensure(idU, is.String);
      const { body, matched } = await searchSnippet(loader, id);
      if (body == null) {
        throw new Error(`Unknown snippet id: ${id}`);
      }
      const bodyStr = is.String(body) ? body : await body(denops, matched);
      const parsedBody = lsputil.parseSnippet(bodyStr);
      if (parsedBody === "") {
        throw new Error(`Failed parsing: ${bodyStr}`);
      }
      return parsedBody;
    },
```